test/statistics_gtest.cc

```diff
@@ -21,8 +21,8 @@ TEST(StatisticsTest, Median) {
 TEST(StatisticsTest, StdDev) {
   EXPECT_DOUBLE_EQ(benchmark::StatisticsStdDev({101, 101, 101, 101}), 0.0);
   EXPECT_DOUBLE_EQ(benchmark::StatisticsStdDev({1, 2, 3}), 1.0);
-  EXPECT_FLOAT_EQ(benchmark::StatisticsStdDev({1.5, 2.4, 3.3, 4.2, 5.1}),
-                  1.42302495);
+  EXPECT_DOUBLE_EQ(benchmark::StatisticsStdDev({2.5, 2.4, 3.3, 4.2, 5.1}),
+                   1.1510864433221342);
```

Comment:
https://travis-ci.org/github/google/benchmark/jobs/668673797 says it's `1.1510864433221341`
Since everything else doesn't agree, i guess this is fast-math discrepancy.
But i also guess this only happened to work before due to the numbers chosen.
Can you use `EXPECT_NEAR()` instead?

Reply:
no need to, i just need to drop the last digit. `EXPECT_DOUBLE_EQ` already checks for within 4 ULP.

Reply:
If that works, sure.